spec/lib/fast_cache/cache_spec.rb

```diff
@@ -112,6 +117,42 @@
         subject[:e].should eq 6
       end
     end
+
+    describe 'cleanup block' do
+      let(:cleanups) { [] }
```

Comment:
How about `before(:each){ cleanups }` to initialize the variable.  Otherwise  `cleanups` doesn't get initialized until after something is the cache callback is invoked in each of the specs.  Since it's single threaded I don't think it matters but something which may cause a headache for the next person working with this.